hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/service/ServiceOperations.java

```diff
@@ -67,26 +66,6 @@ public static Exception stopQuietly(Service service) {
     return stopQuietly(LOG, service);
   }
 
-  /**
-   * Stop a service; if it is null do nothing. Exceptions are caught and
-   * logged at warn level. (but not Throwables). This operation is intended to
-   * be used in cleanup operations
-   *
-   * @param log the log to warn at
-   * @param service a service; may be null
-   * @return any exception that was caught; null if none was.
-   * @see ServiceOperations#stopQuietly(Service)
-   */
-  public static Exception stopQuietly(Log log, Service service) {
```

Comment:
again, we should tag as deprecated in branch 3

Reply:
Unfortunately we can't keep this method on trunk branch, because this method directly takes commons logging Log reference :(
And since this class is IA.Public, this change can only be taken up in trunk branch only and not on branch-3.3 for now. 

We can move forward with log4j2 only on 4.0.0 release (something similar to hbase). FYI @Apache9

Reply:
The problem is, if we deprecate this now, we will have to keep this method. If we keep this method, then we can't cleanup commons-logging from our codebase, because the first arg of the method is still referring to commons-logging logger i.e. `org.apache.commons.logging.Log`.

Reply:
@steveloughran @jojochuang @Apache9 WDYT we should do here?
On one hand, we want to remove commons-logging loggers from code base, that's the main purpose of this PR.
On the other hand, removing commons-logging means we remove this method because it's not in use and it has direct reference to commons-logging logger, and the method is IA.Public.

If we deprecate this method and keep it, we are not done with the purpose of this PR i.e. cleaning up of commons-logging.

Reply:
Done, updated this method as `@Deprecated` with updated javadoc. For now, this PR is good for branch-3.3 backport.

Will create another PR for removing them from trunk.